projects/VRDR/DeathRecord_submissionProperties.cs

```diff
@@ -3869,9 +4191,14 @@ public string UsualOccupation
                 }
                 if (UsualWork == null)
                 {
-                    CreateUsualWork();
+                    CreateUsualWork();  // This does not initialize the value, so it will be NULL, which should be OK.
+
                 }
-                UsualWork.Value = new CodeableConcept(CodeSystems.NullFlavor_HL7_V3, "UNK", "unknown", value);     // code is required
+                // Replace Text field, leave the rest alone.
+                Dictionary<string, string>  usualOccupationDict = CodeableConceptToDict((CodeableConcept)UsualWork.Value);
```

Comment:
When does UsualWork.Value get initialized as a CodeableConcept?

Reply:
It is a codeable concept.
For submissions, only the codeable.text field is used.
For coded responses, the text is returned, and the coding.code is used for the coded value.